docs/docker.md

```diff
@@ -16,17 +42,28 @@ procedure on a regular basis (e.g. via cron). We recommend executing it every
 30 minutes. This is necessary to execute maintenance tasks such as enforcing
 data retention policies according to the configured policy.
 
+**Example:**
+
+When using timescale as the database name:
```

Comment:
Does the database name matter here? It might be a bit confusing as dbname isn't used in cmd at all. Maybe just add the command that needs to be used.